.github/ISSUE_TEMPLATE/---signaturedoesnotmatch-error.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "\U0001F6AB SignatureDoesNotMatch error report"
+about: Report a `SignatureDoesNotMatch` client error.
+title: 'SignatureDoesNotMatch error report'
+labels: needs-triage
+assignees: ''
+
+---
+
+If you have encountered an error message like the following, you're in the right place! If not, please [file a standard bug report](https://github.com/aws/aws-cli/issues/new/choose).
+
+> `A client error (SignatureDoesNotMatch) occurred when calling the ServiceFunction operation: The request signature we calculated does not match the signature you provided. Check your key and signing method.`
+
+where the `ServiceFunction` called changes depending on what command the user ran. For example, if the command issued was `aws ec2 describe-instances`, then the service function would be `describe-instances`.
```

Comment:
We can probably remove the note explaining `ServiceFunction` and just add a placeholder like `<ApiCall>`. The explanation is a little hard to follow.